config/gazebo8_debian.upstream.yaml

```diff
@@ -0,0 +1,6 @@
+name: gazebo8_debian_import
+method: http://packages.osrfoundation.org/gazebo/debian-stable
+suites: [stretch]
+component: main
+architectures: [amd64, arm64, armhf, i386, source]
+filter_formula: Package (% gazebo-dbg ) | Package (% gazebo ) | Package (% sdformat)
```

Comment:
This probably needs gazebo8 specifically as well as any dependencies that need to come from packages.osrfoundation.org like sdformat? ignition? @j-rivero Can you comment what's needed in this import. Like the gazebo5 versions: https://github.com/ros-infrastructure/reprepro-updater/commit/3c333ac5c6b2fedada768e60582eaa2c437079ae